t/op/for-many.t

```diff
@@ -0,0 +1,349 @@
+#!./perl
+
+BEGIN {
+    chdir 't' if -d 't';
+    require "./test.pl";
+}
+
+use strict;
+use warnings;
+
+my @have;
+
```

Comment:
You implement and document continue blocks for the new construct, but don't test them.

Reply:
Well spotted. Fixed.

We also didn't seem to have tests for LVALUE aliases in exsting `for` loops, and for attempting to modify RVALUE aliases --  #19133